connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoClient.scala

```diff
@@ -31,6 +31,10 @@ class KustoClient(val clusterAlias: String, val engineKcsb: ConnectionStringBuil
   lazy val dmClient: Client = ClientFactory.createClient(ingestKcsb)
   lazy val ingestClient: IngestClient = IngestClientFactory.createClient(ingestKcsb)
 
+  private lazy val exportCacheEntryCreator = (c: ContainerAndSas) => KDSU.parseSas(c.containerUrl + c.sas)
+  private lazy val  ingestContainersCache = new ContainersCache(dmClient, clusterAlias, generateCreateTmpStorageCommand(), (c: ContainerAndSas) => c)
```

Comment:
this might be a silly question, but why do we need to split the ingestion containers from the export ones?

Reply:
nvm, noticed the different command